src/hardware/gus.cpp

```diff
@@ -627,11 +665,10 @@ void Gus::ActivateVoices(uint8_t requested_voices)
 		// Gravis' calculation to convert from number of active voices
 		// to playback frame rate. Ref: UltraSound Lowlevel ToolKit
 		// v2.22 (21 December 1994), pp. 3 of 113.
-		frame_rate_per_ms = 1000.0 / (1.619695497 * active_voices);
-
-		frame_rate_hz = iround(frame_rate_per_ms * 1000.0);
+		const auto rate   = 1000000.0 / (1.619695497 * active_voices);
+		gus_frame_rate_hz = iround(rate);
```

Comment:
I've noticed that rounding the frame rate doesn't exactly agree with what's in the Lowlevel Toolkit docs – a few of the rates are high by 1Hz (for example, with 16 voices, it gives 38588 instead of the 38587 in the docs). Taking the floor instead of rounding makes the frame rate agree with all rates shown in the docs.

Reply:
Thank you, @polpo - will update.